FWCore/ParameterSet/python/Config.py

```diff
@@ -1119,17 +1119,16 @@ def __call__(self,obj):
     for k in self.__args.iterkeys():
         if hasattr(obj,k):
             params[k] = getattr(obj,k)
-        else:
-            params[k] = self.__args[k]
     _modifyParametersFromDict(params, self.__args, self._raiseUnknownKey)
     for k in self.__args.iterkeys():
         if k in params:
             setattr(obj,k,params[k])
         else:
             #the parameter must have been removed
             delattr(obj,k)
+  @staticmethod
   def _raiseUnknownKey(key):
```

Comment:
Other option would have been to add `self` parameter, but since it would go unused I chose `@staticmethod`.